internal/compiler/wasm/wasm.go

```diff
@@ -398,6 +403,43 @@ func (c *Compiler) compileStrings() error {
 		c.stringAddrs[i] = addr
 	}
 
+	// interned `opa_value*` for these true/false booleans
```

Comment:
This function has grown quite a bit over time. I think we could factor this section into another function (called from here) for readability.

Reply:
good point! will do.

Reply:
I've split them all off. ✔️